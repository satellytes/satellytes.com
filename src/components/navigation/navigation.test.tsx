import React from 'react';
import Navigation from './navigation';
import { render } from '@testing-library/react';

describe('<Navigation />', () => {
  it('should render', () => {
    const title = 'Test title';
    const { getByText } = render(<Navigation siteTitle={title} />);
    const navigation = getByText(title);
    expect(navigation).toBeInTheDocument();
  });
});
