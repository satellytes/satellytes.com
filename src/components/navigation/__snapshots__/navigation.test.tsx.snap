// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-fubCfw gwtvNU"
      >
        <div
          class="sc-bdfBwQ jtOzOH"
        >
          <div
            class="sc-gsTCUz hfpEbi"
          />
          <div
            class="sc-gsTCUz fExTmY"
          >
            <div
              class="sc-pFZIQ kkbJfl"
            >
              <div
                class="sc-jrAGrp hjHJNJ"
              >
                <ul
                  class="sc-crrsfI hNUHNK"
                >
                  <li
                    class="sc-dQppl JJGup"
                  >
                    <a
                      class="sc-bqyKva dRKGoA"
                      href="https://www.linkedin.com/company/satellytes"
                      rel="noopener noreferrer"
                      target="_blank"
                      title="Go to the Satellytes LinkedIn profile"
                    >
                      <svg
                        class="sc-dlfnbm iAMZkD"
                        fill="none"
                        height="22"
                        viewBox="0 0 22 22"
                        width="22"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M21.9947 22V21.9991H22.0002V13.9726C22.0002 10.046 21.147 7.0213 16.5136 7.0213C14.2862 7.0213 12.7914 8.23229 12.1812 9.38036H12.1168V7.38788H7.72363V21.9991H12.2981V14.7642C12.2981 12.8592 12.6626 11.0172 15.0437 11.0172C17.3898 11.0172 17.4248 13.1912 17.4248 14.8864V22H21.9947Z"
                        />
                        <path
                          d="M0.467773 7.25531H4.91458V22H0.467773V7.25531Z"
                        />
                        <path
                          d="M2.69149 0C1.20566 0 0 1.20004 0 2.67894C0 4.15784 1.20566 5.38297 2.69149 5.38297C4.17731 5.38297 5.38297 4.15784 5.38297 2.67894C5.38204 1.20004 4.17638 0 2.69149 0V0Z"
                        />
                      </svg>
                    </a>
                  </li>
                  <li
                    class="sc-dQppl JJGup"
                  >
                    <a
                      class="sc-bqyKva dRKGoA"
                      href="https://www.xing.com/companies/satellytesgmbh"
                      rel="noopener noreferrer"
                      target="_blank"
                      title="Go to the Satellytes Xing profile"
                    >
                      <svg
                        class="sc-hKgILt hoWFJd"
                        fill="none"
                        height="22"
                        viewBox="0 0 21 22"
                        width="21"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M20.5 0H15.3704L7.75 13.3746L12.6601 22H17.7897L12.8796 13.3746L20.5 0Z"
                        />
                        <path
                          d="M5.7055 4.25H0.875766L3.67027 9.07541L0 15H4.82973L8.5 9.07541L5.7055 4.25Z"
                        />
                      </svg>
                    </a>
                  </li>
                  <li
                    class="sc-dQppl JJGup"
                  >
                    <a
                      class="sc-bqyKva dRKGoA"
                      href="https://github.com/satellytes"
                      rel="noopener noreferrer"
                      target="_blank"
                      title="Go to the Satellytes Github profile"
                    >
                      <svg
                        class="sc-eCssSg htElcU"
                        height="24"
                        viewBox="0 0 24 24"
                        width="24"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z"
                        />
                      </svg>
                    </a>
                  </li>
                </ul>
                <ul
                  class="sc-kEjbxe fpgicB"
                >
                  <li
                    class="sc-iqHYGH esUSCK"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-kstrdz dhyhWs"
                      href="/imprint"
                    >
                      Impressum
                    </a>
                  </li>
                  <li
                    class="sc-iqHYGH esUSCK"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-kstrdz dhyhWs"
                      href="/data-privacy"
                    >
                      Datenschutz
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="sc-hBEYos fiUsv"
              >
                <span
                  class="sc-fodVxV doEYUN"
                >
                  Menu
                </span>
                <nav>
                  <ul
                    class="sc-fFubgz eLdEqB"
                  >
                    <li
                      class="sc-bkzZxe hXrXUW"
                    >
                      <a
                        class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                        href="/services"
                      >
                        Leistungen
                      </a>
                    </li>
                    <li
                      class="sc-bkzZxe hXrXUW"
                    >
                      <a
                        class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                        href="/clients"
                      >
                        Kunden
                      </a>
                    </li>
                    <li
                      class="sc-bkzZxe hXrXUW"
                    >
                      <a
                        class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                        href="/career"
                      >
                        Karriere
                      </a>
                    </li>
                    <li
                      class="sc-bkzZxe hXrXUW"
                    >
                      <a
                        class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                        href="/contact"
                      >
                        Kontakt
                      </a>
                    </li>
                  </ul>
                </nav>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-fubCfw gwtvNU"
    >
      <div
        class="sc-bdfBwQ jtOzOH"
      >
        <div
          class="sc-gsTCUz hfpEbi"
        />
        <div
          class="sc-gsTCUz fExTmY"
        >
          <div
            class="sc-pFZIQ kkbJfl"
          >
            <div
              class="sc-jrAGrp hjHJNJ"
            >
              <ul
                class="sc-crrsfI hNUHNK"
              >
                <li
                  class="sc-dQppl JJGup"
                >
                  <a
                    class="sc-bqyKva dRKGoA"
                    href="https://www.linkedin.com/company/satellytes"
                    rel="noopener noreferrer"
                    target="_blank"
                    title="Go to the Satellytes LinkedIn profile"
                  >
                    <svg
                      class="sc-dlfnbm iAMZkD"
                      fill="none"
                      height="22"
                      viewBox="0 0 22 22"
                      width="22"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M21.9947 22V21.9991H22.0002V13.9726C22.0002 10.046 21.147 7.0213 16.5136 7.0213C14.2862 7.0213 12.7914 8.23229 12.1812 9.38036H12.1168V7.38788H7.72363V21.9991H12.2981V14.7642C12.2981 12.8592 12.6626 11.0172 15.0437 11.0172C17.3898 11.0172 17.4248 13.1912 17.4248 14.8864V22H21.9947Z"
                      />
                      <path
                        d="M0.467773 7.25531H4.91458V22H0.467773V7.25531Z"
                      />
                      <path
                        d="M2.69149 0C1.20566 0 0 1.20004 0 2.67894C0 4.15784 1.20566 5.38297 2.69149 5.38297C4.17731 5.38297 5.38297 4.15784 5.38297 2.67894C5.38204 1.20004 4.17638 0 2.69149 0V0Z"
                      />
                    </svg>
                  </a>
                </li>
                <li
                  class="sc-dQppl JJGup"
                >
                  <a
                    class="sc-bqyKva dRKGoA"
                    href="https://www.xing.com/companies/satellytesgmbh"
                    rel="noopener noreferrer"
                    target="_blank"
                    title="Go to the Satellytes Xing profile"
                  >
                    <svg
                      class="sc-hKgILt hoWFJd"
                      fill="none"
                      height="22"
                      viewBox="0 0 21 22"
                      width="21"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M20.5 0H15.3704L7.75 13.3746L12.6601 22H17.7897L12.8796 13.3746L20.5 0Z"
                      />
                      <path
                        d="M5.7055 4.25H0.875766L3.67027 9.07541L0 15H4.82973L8.5 9.07541L5.7055 4.25Z"
                      />
                    </svg>
                  </a>
                </li>
                <li
                  class="sc-dQppl JJGup"
                >
                  <a
                    class="sc-bqyKva dRKGoA"
                    href="https://github.com/satellytes"
                    rel="noopener noreferrer"
                    target="_blank"
                    title="Go to the Satellytes Github profile"
                  >
                    <svg
                      class="sc-eCssSg htElcU"
                      height="24"
                      viewBox="0 0 24 24"
                      width="24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z"
                      />
                    </svg>
                  </a>
                </li>
              </ul>
              <ul
                class="sc-kEjbxe fpgicB"
              >
                <li
                  class="sc-iqHYGH esUSCK"
                >
                  <a
                    class="sc-jSgupP iinwFt sc-kstrdz dhyhWs"
                    href="/imprint"
                  >
                    Impressum
                  </a>
                </li>
                <li
                  class="sc-iqHYGH esUSCK"
                >
                  <a
                    class="sc-jSgupP iinwFt sc-kstrdz dhyhWs"
                    href="/data-privacy"
                  >
                    Datenschutz
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="sc-hBEYos fiUsv"
            >
              <span
                class="sc-fodVxV doEYUN"
              >
                Menu
              </span>
              <nav>
                <ul
                  class="sc-fFubgz eLdEqB"
                >
                  <li
                    class="sc-bkzZxe hXrXUW"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                      href="/services"
                    >
                      Leistungen
                    </a>
                  </li>
                  <li
                    class="sc-bkzZxe hXrXUW"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                      href="/clients"
                    >
                      Kunden
                    </a>
                  </li>
                  <li
                    class="sc-bkzZxe hXrXUW"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                      href="/career"
                    >
                      Karriere
                    </a>
                  </li>
                  <li
                    class="sc-bkzZxe hXrXUW"
                  >
                    <a
                      class="sc-jSgupP iinwFt sc-idOhPF fyEDMC"
                      href="/contact"
                    >
                      Kontakt
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
